MapboxAndroidDemo/src/main/res/layout/activity_location_basic.xml

```diff
@@ -21,7 +21,8 @@
         android:layout_height="wrap_content"
         android:layout_gravity="end|bottom"
         android:layout_margin="16dp"
-        android:src="@drawable/ic_my_location_24dp"
+        app:srcCompat="@drawable/ic_my_location_24dp"
```

Comment:
It may be an issue specific to the Nexus 5, Android 6.0.1, API 23.  `app:srcCompat` was the only way I could get it to work on a device.